scalafix-core/shared/src/main/scala/scalafix/internal/rule/DisableVariantTypes.scala

```diff
@@ -0,0 +1,26 @@
+package scalafix.internal.rule
+
+import scalafix.rule.RuleCtx
+import scalafix.{LintMessage, SemanticRule, SemanticdbIndex}
+import scala.meta._
+import scalafix.lint.LintCategory
+
+final case class DisableVariantTypes(index: SemanticdbIndex)
+    extends SemanticRule(index, "DisableVariantTypes") {
+
+  private lazy val errorCategory: LintCategory =
+    LintCategory.error(
+      """Covariant and Contravariant type parameters are disabled""".stripMargin
```

Comment:
This definitely deserves a longer explanation text, since it's not clear why it's disabled.